cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -59,33 +78,35 @@ else
   exit 1
 fi
 
-# Enable and restart munge service
-systemctl enable munge
-echo "Restarting munge service"
-systemctl restart munge
+if [ "$CONFIG_PROCESS" == "False" ]; then
```

Comment:
Yesterday we discussed about this and we just said:
- if the script is called without any option, it will check the status of compute (and login) fleet (that's how users should use it directly on the head node).
- if the script is passed with the boolean flag, this flag disables the check of the resources (this is what we are going to do in our recipes, since in that case we are already covered by:
  - the fact that such resources do not exist yet at cluster creation time;
  - the update policies at cluster update time.
  
I'm sorry if there was any misunderstanding about this.  

The way this is implemented now, the script is just going to be completely disabled by the boolean flag (and we don't want that).

Reply:
Done as we discuss today.